### DISCUSSION TITLE
Add shadow to sprite thumbnail in code area

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6577] [Ideas]
When in your project you have a white sprite (e.g. the digits for a game score) the sprite thumbnail in the code area is not visible. It would help adding a shadow to the sprite thumbnail in the code area.

**Comment by DNin01** [comment]
It's a nice idea but the addon would be so small and hard to describe.

**Reply by WorldLanguages** [reply]
Would the shadow always show, or only when clicking the sprite in the sprite panel?

**Reply by BroJac5246** [reply]
> Would the shadow always show, or only when clicking the sprite in the sprite panel?

Maybe it could be a setting:
Always | On hover | When selected

**Reply by sfederici** [reply]
My idea is that the shadow will be always shown if the addon is enabled

**Reply by WorldLanguages** [reply]
Oh, I see, you're saying the `sprite thumbnail in the code area`

**Reply by sfederici** [reply]
Yes, exactly

**Reply by WorldLanguages** [reply]
I didn't know this had a name:

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/674d32f5-abd3-41bf-944e-2092b6260e7b)

**Reply by CST1229** [reply]
I think the developers call this the sprite/stage watermark (at least, according to https://github.com/scratchfoundation/scratch-gui/pull/3342).
